doc-content/shared-kie-docs/src/main/asciidoc/Workbench/ArchetypeManagement/proc-business-central-additional-features-archetype.adoc

```diff
@@ -9,7 +9,7 @@ You can delete, set a default, and validate the archetypes from the *Archetypes*
 
 .Procedure
 
-. In {CENTRAL}, select the *Admin* icon in the top-right corner of the screen and select *Archetypes*.
+. In {CENTRAL}, select the *Admin* icon in the upper-right corner of the screen and select *Archetypes*.
```

Comment:
```suggestion
. In {CENTRAL}, select the *Admin* icon and then select *Archetypes*.
```
[Style Guide - Location of interface elements](https://source.redhat.com/groups/public/ccs/ccs_folder/ibm_style_guide/ibm_style_062121_q2_2021pdf) - "When you refer to the location of an element in a graphical user interface, do not use sensory words, such as top, bottom, right, or left. Instead of sensory words, refer to the element by its label or tooltip"

Reply:
Noted